balancer-labs/frontend-v2
# Trade UI cleanup

# Description

Adds in a few quality of life changes to the trade UI.
1. People have complained about the price graph being 'inverse' so, it's been inversed to un-inverse the way the users would see it.
2. Add in a small icon tooltip to indicate where the pricing data for the graph is coming from.

## Type of change


## How should this be tested?

Ensure that the pair price graph is showing the pricing data in terms of `outputAsset/inputAsset` and in label too
Ensure that there is a small coingecko icon next to the pair symbols on the price graph and has a tooltip

## Checklist:


## Walkthrough
https://www.loom.com/share/b43bbcdc2fbf45c1831ae701f485581d
